### PR TITLE
Quickfix

### PR DESCRIPTION
**Pull request by opotowsky**

### CODE REVIEW

**Comment by opotowsky** [comment]
Pinging @gidden @rwcarlsen @mbmcgarry @scopatz for review; not sure if this is an appropriate solution for 1.3.newer!

File: genapi.py [hunk 1]
```diff
@@ -908,12 +908,13 @@ def parse_args(argv):
 def setup(ns):
     """Ensure that we are ready to perform code generation. Returns typesystem."""
     # load raw table
-    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
+    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
```
Comment (gidden) [inline genapi.py]: is there a reason this has to be in the src_dir? why not hold the .js file and write the json file to the build dir? that would keep things "mostly" the same.

File: genapi.py [hunk 2]
```diff
@@ -908,12 +908,13 @@ def parse_args(argv):
...
     if not os.path.exists(ns.build_dir):
         os.mkdir(ns.build_dir)
     if not os.path.isfile(dbtypes_json):
-        print('Downloading ' + DBTYPES_JS_URL + ' ...')
-        f = urlopen(DBTYPES_JS_URL)
+        #print('Downloading ' + DBTYPES_JS_URL + ' ...')
```
Comment (opotowsky) [inline genapi.py]: @gidden to double-check, is this acceptable to leave in the commented out code or should I delete it for the PR?

File: genapi.py (hunk 1 above)
Comment (opotowsky) [inline genapi.py]: I guess I assumed that the .js and .json files should be together, but that was probably silly to assume! I'll update it to be created in the build directory.

File: genapi.py (hunk 2 above)
Comment (gidden) [inline genapi.py]: Remove it, please. Also, would you make an issue (if there isn't already
one) covering this? Basically outlining the problem and perhaps proposing a
more long-term solution.

On Fri, Jun 5, 2015 at 1:02 PM, opotowsky <notifications@github.com> wrote:

> In genapi.py
> <https://github.com/cyclus/cymetric/pull/77#discussion_r31836163>:
>
> >      if not os.path.exists(ns.build_dir):
> >          os.mkdir(ns.build_dir)
> >      if not os.path.isfile(dbtypes_json):
> > -        print('Downloading ' + DBTYPES_JS_URL + ' ...')
> > -        f = urlopen(DBTYPES_JS_URL)
> > +        #print('Downloading ' + DBTYPES_JS_URL + ' ...')
>
> @gidden <https://github.com/gidden> to double-check, is this acceptable
> to leave in the commented out code or should I delete it for the PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cymetric/pull/77/files#r31836163>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

File: genapi.py (hunk 1 above)
Comment (gidden) [inline genapi.py]: The .js file is input and the json file is the result of some processing
(and can be easily regenerated).

On Fri, Jun 5, 2015 at 1:03 PM, opotowsky <notifications@github.com> wrote:

> In genapi.py
> <https://github.com/cyclus/cymetric/pull/77#discussion_r31836270>:
>
> > @@ -908,12 +908,13 @@ def parse_args(argv):
> >  def setup(ns):
> >      """Ensure that we are ready to perform code generation. Returns typesystem."""
> >      # load raw table
> > -    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
> > +    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
>
> I guess I assumed that the .js and .json files should be together, but
> that was probably silly to assume! I'll update it to be created in the
> build directory.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cymetric/pull/77/files#r31836270>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

File: genapi.py (hunk 2 above)
Comment (opotowsky) [inline genapi.py]: I'll make a new issue to reflect our discussion last week and am on the update

Comment (opotowsky) [inline genapi.py]: Rather, I will update the current one since it discusses the relevant cymetric bug

**Comment by scopatz** [comment]
This is not a good fix because it duplicated information across different sources in the ecosystem. Preferably they should be only one canonical location (aside from the source code itself) where this information is garnered from. The correct fix is in cyclus/cyclus.github.com#157

**Comment by opotowsky** [comment]
Closing since https://github.com/cyclus/cymetric/pull/79, https://github.com/cyclus/cymetric/pull/80, and https://github.com/cyclus/cymetric/pull/81 fix this issue in a better way

**Pull request closed without merging**